libs/blocks/src/lib/features/content-left/index.tsx

```diff
@@ -1,6 +1,7 @@
 import { FluidContainer, Heading, Section, Text } from '@deriv/quill-design';
 import clsx from 'clsx';
 import React, { ReactNode } from 'react';
+import { container, contentContainer } from './content-left.classnames';
```

Comment:
Let's add `Variant` or `CVA` in the name of these `CVA`s otherwise they will look like a component or some simple variables.

Reply:
Done with adding CVA to the name